data/beaconrestapi/src/integration-test/resources/tech/pegasys/teku/beaconrestapi/beacon/schema/Bytes20.json

```diff
@@ -0,0 +1,8 @@
+{
+  "type" : "object",
+  "properties" : {
+    "wrappedBytes" : {
+      "$ref" : "#/components/schemas/Bytes"
+    }
+  }
+}
```

Comment:
Why are we getting this object? Doesn't seem like something we should have. We may need to just use `Bytes` as the type in json API definitions rather than `Bytes20`, though I'm surprised this is turning up now given we were already using `Bytes20` for coinbase.